### PR TITLE
Oriolperramon8396/sc 2023/clicking a paragraph not working video

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: libs/search-widget/src/viewer/previewers/Pdf.svelte [hunk 1]
```diff
@@ -110,15 +110,13 @@
 
   const find = (query: string) => {
     // TODO: pdf search is still not reliable enough
```
Comment (mpellerin42) [inline libs/search-widget/src/viewer/previewers/Pdf.svelte:112]: should we remove this TODO now the code is uncommented?

File: libs/search-widget/src/viewer/store.ts [hunk 2]
```diff
@@ -447,9 +448,15 @@ function findParagraphFromSearchSentence(
...
     strict
-      ? paragraph.sentences?.find((sentence) => searchSenctence.text === getSentenceText(field!, sentence))
-      : (getParagraphText(field!, paragraph) || '').includes(searchSenctence.text.trim()),
+      ? paragraph.sentences?.find((sentence) => text === getSentenceText(field!, sentence))
+      : (getParagraphText(field!, paragraph) || '').includes(text.trim()),
   );
 }
+
+function normalizeSearchParagraphText(text: string) {
+  return text.replace('<mark>','').replace('</mark>', '');
```
Comment (mpellerin42) [inline libs/search-widget/src/viewer/store.ts:460]: This will replace only the first occurence of `<mark>` and `</mark>`. Is this in purpose?

Comment (operramon) [reply libs/search-widget/src/viewer/store.ts]: I didn't notice. It's fixed

File: libs/search-widget/src/viewer/previewers/Pdf.svelte (hunk 1 above)
Comment (operramon) [reply libs/search-widget/src/viewer/previewers/Pdf.svelte:112]: In this case it's in purpose, pdf search still need more work.

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**